src/backends/tensorrt/plan_backend.cc

```diff
@@ -361,32 +390,54 @@ PlanBackend::CreateExecutionContexts(
                                             ": " + cudaGetErrorString(cuerr));
           }
 
-          RETURN_IF_ERROR(LoadPlan(
-              mn_itr->second, &eit->second.first, &eit->second.second));
+          nvinfer1::ICudaEngine* shared_engine = nullptr;
+          if (dla_core_id != -1) {
+            auto device_pair = std::make_pair(gpu_device, dla_core_id);
```

Comment:
Do we really need to have gpu_engines_ and dla_engines_ as separate maps? Can't we just have device_engines_ that represents the gpu/dla pair directly. That seems much easier to understand and manage.

Reply:
Done, merged into a single device_engines_ map